src/content/eol/2024/08/eol-08-14-24.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Change to user and license keys"
+summary: "On September 25, 2024, we're removing the ability to copy user and license API keys from the UI."
+publishDate: '2024-08-01'
+eolEffectiveDate: '2024-09-09'
+---
+
+Beginning September 25, 2024, we are removing the ability to copy API keys from [the UI](https://get.newrelic.com/MzQxLVhLUC0zMTAAAAGUiZ8sq6uwzvQMEjtdfLs43rOhHFRYwkHxXjt8wYs2qCJP2IkoVpEZiGhCVneo-hfy1Ucijgo=). The “copy key” option in the drop-down seen below will be removed. This applies to user and license keys, both new and existing. If you have an automation that is dependent on scraping keys from the UI, please integrate using the NerdGraph API.
+
+You are still able to retrieve your new and existing API keys through our [NerdGraph API](https://docs.newrelic.com/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys/#query-keys).
+
+## What you need to do
+
+Before **September 25, 2024**: you will need to store your existing API keys (both user and license keys) in a secure location outside New Relic, if you haven't done that already. **If you do not take this step, you will no longer be able to access existing API keys from our UI.**
```

Comment:
I think the "y" in "you will" should be capitalized since it follows the colon, right? Also you will > you'll

"haven't done that" I think is more commonly "haven't done so", but this is splitting the smallest of hairs

do not > don't, followed by you will > you'll. Also we should probably rewrite the last clause to "you'll lose access to your existing API keys from our UI." to remove the passive voice, what do you think?